AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_LightAdded.py

```diff
@@ -97,74 +182,299 @@ def AtomEditorComponents_Light_AddedToEntity():
+            light_color = light_component.get_component_property_value(
+               AtomComponentProperties.light('Color'))
+            general.idle_wait_frames(1)
+            Report.result(Tests.edit_light_color, light_color.IsClose(color_value))
+
+            # 9. Set Intensity mode.
+            # if current_light_type < 6:
+            #     for intensity_mode in INTENSITY_MODE.keys():
+            #         light_component.set_component_property_value(
+            #             AtomComponentProperties.light('Intensity mode'), INTENSITY_MODE[intensity_mode])
+            #         current_intensity_mode = light_component.get_component_property_value(
+            #             AtomComponentProperties.light('Intensity mode'))
+            #         test_intensity_mode = (
+            #             f"Intensity mode set to {current_intensity_mode}",
+            #             f"P1: Intensity mode failed to be set to {current_intensity_mode}")
+            #         Report.result(test_intensity_mode,
+            #                       current_intensity_mode == INTENSITY_MODE[intensity_mode])
+
+            # 10. Edit the Intensity parameter.
+            light_component.set_component_property_value(AtomComponentProperties.light('Intensity'), 1000)
+            Report.result(Tests.edit_intensity_value,
+                          light_component.get_component_property_value(
+                              AtomComponentProperties.light('Intensity')) == 1000)
+
+            # 11. Set the Attenuation radius Mode:
+            for radius_mode in ATTENUATION_RADIUS_MODE.keys():
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Attenuation radius Mode'), ATTENUATION_RADIUS_MODE[radius_mode])
+                current_radius_mode = light_component.get_component_property_value(
+                   AtomComponentProperties.light('Attenuation radius Mode'))
+                test_attenuation_mode = (
+                    f"Attenuation radius Mode set to {radius_mode}",
+                    f"P1: Attenuation radius Mode failed to be set to {radius_mode}")
+                general.idle_wait_frames(1)
+                Report.result(test_attenuation_mode,
+                              current_radius_mode == ATTENUATION_RADIUS_MODE[radius_mode])
+
+                # 12. Edit the Attenuation radius Radius parameter.
+                if current_radius_mode == 0:
+                    light_component.set_component_property_value(
+                        AtomComponentProperties.light('Attenuation radius Radius'), 1000)
+                    Report.result(Tests.edit_attenuation_radius,
+                                  light_component.get_component_property_value(
+                                      AtomComponentProperties.light('Attenuation radius Radius')) == 1000)
+
+            # Shadow tests for applicable light types:
+            if current_light_type in (1, 2):
```

Comment:
same here use `LIGHT_TYPE` dict reference for the number

Reply:
changed to:
if light_type in ('sphere', 'spot_disk'):